### PR TITLE
Feature/ab2d 1033 load properties

### PR DESCRIPTION
**Pull request by adaykin**
Move several properties to the database so that they can be reloaded

Create a migration to create a table for storing properties and insert 3 that will be able to be reloaded. On application startup the properties will be read from the database and then set so that places throughout the application that access properties can use the conventional way of accessing them in Spring.

### Checklist

- [x] Tests and linting pass
- [x] Code checked for PHI/PII exposure

### Security
What secure items does this touch, how does this PR affect our security posture, etc?

NA

### CODE REVIEW

File: common/src/main/resources/db/changelog/v001/add_properties_table.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    key VARCHAR(80) NOT NULL,
+    value VARCHAR(255) NOT NULL
+);
+
+ALTER TABLE properties ADD CONSTRAINT "uc_properties_key" UNIQUE (key);
+ALTER TABLE properties ADD CONSTRAINT "pk_properties" PRIMARY KEY (id);
+
+INSERT INTO properties (id, key, value) VALUES((select nextval('hibernate_sequence')), 'pcp.core.pool.size', 10);
+INSERT INTO properties (id, key, value) VALUES((select nextval('hibernate_sequence')), 'pcp.max.pool.size', 150);
+INSERT INTO properties (id, key, value) VALUES((select nextval('hibernate_sequence')), 'pcp.scaleToMax.time', 900);
```
Comment (spathiyil) [inline common/src/main/resources/db/changelog/v001/add_properties_table.sql]: Rollback?

File: worker/src/test/java/gov/cms/ab2d/worker/config/AutoScalingServiceTest.java [hunk 2]
```diff
@@ -97,7 +97,7 @@ void autoScalingKicksInAndResizes() throws InterruptedException {
         // Last metric taken should always be MAX_POOL_SIZE
         assertThat(new ArrayDeque<>(metrics).getLast(), equalTo(MAX_POOL_SIZE));
         // Some intermediate sizes should be in between, at least 3.
-        assertThat(metrics.size(), greaterThanOrEqualTo(4));
+        assertThat(metrics.size(), greaterThanOrEqualTo(3));
```
Comment (dkrylovsb) [inline worker/src/test/java/gov/cms/ab2d/worker/config/AutoScalingServiceTest.java]: The comment above is a bit confusing, I agree. 3 intermediate metrics plus one final is 4 total; so we should keep `4`. Let's update the comment instead.

File: common/src/main/resources/db/changelog/v001/add_properties_table.sql (hunk 1 above)
Comment (adaykin) [reply common/src/main/resources/db/changelog/v001/add_properties_table.sql]: I can drop the table

Comment (spathiyil) [reply common/src/main/resources/db/changelog/v001/add_properties_table.sql]: okay. 
Am not sure, but it is possible you may need to drop the unique constraint `uc_properties_key` as well, if it does not get automatically deleted when you drop the table.

Comment (adaykin) [reply common/src/main/resources/db/changelog/v001/add_properties_table.sql]: I think it's only a foreign key reference that would prevent deletion

Comment (spathiyil) [reply common/src/main/resources/db/changelog/v001/add_properties_table.sql]: okay

File: worker/Dockerfile [hunk 3]
```diff
@@ -1,4 +1,10 @@
...
+
 FROM openjdk:13
-WORKDIR /usr/src/ab2d-worker
-ADD target /usr/src/ab2d-worker
-CMD java -javaagent:/usr/src/ab2d-worker/newrelic/newrelic.jar -jar worker-*-SNAPSHOT.jar
\ No newline at end of file
+VOLUME /tmp
+ARG JAR_FILE=target/worker-*-SNAPSHOT.jar
+COPY ${JAR_FILE} app.jar
+ENTRYPOINT ["java","-Djava.security.egd=file:/dev/./urandom","-jar","/app.jar"]
```
Comment (dkrylovsb) [inline worker/Dockerfile]: I think this is left-overs from debugging CI; we can revert

**Pull request merged**